spec/lib/va_profile/profile_information/service_spec.rb

```diff
@@ -0,0 +1,858 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'va_profile/profile_information/service'
+
+describe VAProfile::ProfileInformation::Service, :skip_vet360 do
+  subject { described_class.new(user) }
+
+  let(:user) { build(:user, :loa3) }
+  let(:vet360_id) { '1' }
+
+  before do
+    allow(user).to receive_messages(vet360_id:, icn: '1234')
+    Flipper.enable(:va_profile_information_v3_service)
+  end
+
+  after do
+    Flipper.disable(:va_profile_information_v3_service)
+  end
+
+  # describe '#get_person' do
+  # context 'when successful' do
+  #   it 'returns a status of 200' do
+  #     VCR.use_cassette('va_profile/profile_information/person_full', VCR::MATCH_EVERYTHING) do
+  #       response = subject.get_response
+  #       expect(response).to be_ok
+  #       expect(response.person).to be_a(VAProfile::Models::Person)
+  #     end
+  #   end
```

Comment:
I see a lot of tests in this file are commented out. That doesn't seem good 😰

Reply:
@RachalCassity Can you explain all the commented out tests

Reply:
@LindseySaari Those will be uncommented in the next PR.